src/rpc/http2.md

```diff
@@ -0,0 +1,56 @@
+# HTTP/2
+
+HTTP/2 evolved out of Google's experimental SPDY protocol as a successor to HTTP/1.1. Since it maintains high-level compatability with HTTP/1.1 let's first take a look at HTTP/1.x.

```

Comment:
```suggestion
HTTP/2 evolved out of Google's experimental SPDY protocol as a successor to HTTP/1.1. Since it maintains high-level compatibility with HTTP/1.1, let's first take a look at HTTP/1.x.
```